public/javascripts/Gallery/src/modal/Modal.js

```diff
@@ -84,6 +89,26 @@ function Modal(uiModal) {
         attachEventHandlers();
     }
 
+    /**
+     * Fires when a keydown event occurs. Removed after modal closes.
+     */
+    function keydownListener(event) {
```

Comment:
Can you actually create a new `Keyboard.js` file, similar to the one for SVValidate (And SVLabel)? Then I think that most of the keyboard-related stuff should go in there! Since we're finally adding shortcuts, may as well keep it consistent with our other tools!

Reply:
Is it okay if I expose the onClickOrKeyDown function and then call it from the newly created Keyboard.js file or should I move all the logic, including the http request, to the Keyboard.js file?

Reply:
I think that calling `onClickOrKeyPress` from `Keyboard.js` is best! Ideally `Keyboard.js` should be doing very little other than logging key presses and calling the appropriate functions elsewhere. Don't want too much of the logic in there :)